### PR TITLE
Fix component purls empty issue

### PR DESCRIPTION
**Pull request by hanchuntao**

### CODE REVIEW

File: openlcs/openlcs/settings.py [hunk 1]
```diff
@@ -273,8 +273,11 @@
 EMAIL_REALM = 'REDHAT.COM'
 
 # Give superuser permission to these users.
-OPENLCS_ADMIN_LIST = ['jzhao', 'yuwang', 'huiwang', 'qduanmu', 'yulwang',
-                      'chhan', 'wbu', 'axuan']
+OPENLCS_ADMIN_LIST = [
+    'jzhao', 'yuwang', 'huiwang', 'qduanmu', 'yulwang', 'chhan', 'axuan',
+    'openlcs-dev-worker01', 'openlcs-qe-worker01', 'openlcs-ci-worker01',
+    'openlcs-stage-worker01', 'openlcs-prod-worker01'
```
Comment (qduanmu) [inline openlcs/openlcs/settings.py:279]: We don't have to update the settings here, and it's weird to have dev/qe/ci/stage workers as admin of prod instance, you could update it in ansible playbook for each env:
https://gitlab.cee.redhat.com/pelc/openlcs-playbooks/-/blame/main/inventory/group_vars/openlcs.yml#L65
Note: use a user list is not a good idea, see OLCS-484.

Comment (hanchuntao) [reply openlcs/openlcs/settings.py]: I don't confrim change https://gitlab.cee.redhat.com/pelc/openlcs-playbooks/-/blame/main/inventory/group_vars/openlcs.yml#L65 can work, becuase in the code, https://github.com/RedHatProductSecurity/openlcs/blob/606e2d7ef944c6bf7fe1585c66e563aecd24e0af/openlcs/authentication/middleware.py#L67, we check this OPENLCS_ADMIN_LIST directly

Comment (hanchuntao) [reply openlcs/openlcs/settings.py:279]: Or we can fix the user management issue in OLCS-484

Comment (qduanmu) [reply openlcs/openlcs/settings.py:279]: Np, ubi8 has a higher priority, let's track in 484.

**Comment by qduanmu** [comment]
LGTM

**Pull request merged**